exchanges/stream/websocket_types.go

```diff
@@ -50,12 +50,12 @@ type Websocket struct {
 	m                            sync.Mutex
 	connector                    func() error
 
-	// ConnectionManager contains the connection candidates and the current
+	// connectionWrapper contains the connection wrappers and the current
```

Comment:
Did you mean connectionManager?


Also: I don't think this comment adds any useful information about this _private_ variable :)

Reply:
yep